codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -73,8 +74,11 @@ public HttpContentCompressor(int compressionLevel) {
      *        state.  {@code 1} uses minimum memory and {@code 9} uses maximum
      *        memory.  Larger values result in better and faster compression
      *        at the expense of memory usage.  The default value is {@code 8}
+     * @param compressThreshold
+     *        The response body is compressed,
+     *        when the response body exceeds the threshold
      */
-    public HttpContentCompressor(int compressionLevel, int windowBits, int memLevel) {
+    public HttpContentCompressor(int compressionLevel, int windowBits, int memLevel, int compressThreshold) {
```

Comment:
We need to keep the existing constructor to not break the API. Please just create an extra constructor and delegate to it.